cmd/boulder-ca/main.go

```diff
@@ -165,6 +166,10 @@ func main() {
 		cmd.Fail("Error in CA config: MaxNames must not be 0")
 	}
 
+	if c.CA.LifespanOCSP.Duration == 0 {
+		c.CA.LifespanOCSP.Duration = 96 * time.Hour
+	}
```

Comment:
This doesn't really matter _that_ much because <0 would still trigger the error, but:

```suggestion
	if c.CA.LifespanOCSP.Duration <= 0 {
		c.CA.LifespanOCSP.Duration = 96 * time.Hour
	}
```

Reply:
Yeah, I specifically did this as `==` because I want this stanza to only be a "supply a default value if they haven't supplied one". Other code (in the library) is responsible for making sure that the supplied value (whether default or otherwise) is reasonable.